docs/resources-overview.md

```diff
@@ -12,24 +9,20 @@ which they form.
 
 ## Dependencies
 
-Knative Serving depends on two other projects in order to function:
-[Istio][istio] and the [Build CRD][build-crd]. Istio is responsible for setting
-up the network routing both inside the cluster and ingress into the cluster. The
-Build CRD provides a custom resource for Kubernetes which provides an extensible
-primitive for creating container images from various sources, for example a Git
-repository.
-
-You can find out more about both from their respective websites.
+Knative Serving depends on Istio[istio] in order to function. Istio is responsible for
+setting up the network routing both inside the cluster and ingress into the cluster.
+You can find out more about it from their website:
```

Comment:
Cut this line, since the link is actually with the [Istio](istio) link above (which isn't done quite correctly...)

Reply:
Removed